collector/diskstats_linux.go

```diff
@@ -257,6 +258,16 @@ func NewDiskstatsCollector(logger log.Logger) (Collector, error) {
 				), valueType: prometheus.GaugeValue,
 			},
 		},
+		queueDescs: []typedFactorDesc{
+			{
+				desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, diskSubsystem, "rotational"),
```

Comment:
Wondering if we should put that into the info metric as label instead. @SuperQ wdyt?

Reply:
I don't have a strong opinion, but I noticed an different instance where a `device` attribute was [defined as a metric](https://github.com/prometheus/node_exporter/blob/acb36765b4bd904ddf7bd3d29a187dfb7aeddbb7/collector/cpu_linux.go#L131) and left out of the `info` metric (as a label).

Reply:
Good question, it depends a bit on how this is to be used. I kinda feel like this would be more useful as an info metric, as you will want to use it on a join with other metrics.

Having it be a separate metric requires some more awkward PromQL.

Reply:
@SuperQ Just to be clear, do you mean exposing this as an `info` metric rather than a `gauge`, as it is done currently?